### PR TITLE
Setup jest

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
We can't use [story shots](https://github.com/storybooks/storybook/tree/master/addons/storyshots) because of other plugins we use. Maybe we
should take the time to decide if
a) this is important to us and
b) we can find a way around this

Closes DCOS-21785

### CODE REVIEW

File: components/badge/__tests__/badge.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from "react";
```
Comment (weblancaster) [inline components/badge/__tests__/badge.tsx]: Can you rename the file to `badge.test.tsx` please?

Comment (DanielMSchmidt) [reply components/badge/__tests__/badge.tsx]: feel free to do it yourself 👍 (because I am OOO for today ;))

Comment (weblancaster) [reply components/badge/__tests__/badge.tsx]: I will do, thanks.

**Comment by weblancaster** [comment]
@DanielMSchmidt let's not use `story shots` for now maybe one day when we find more value but at the moment we are pretty good with Jest Snapshot/Unit tests

**Comment by DanielMSchmidt** [comment]
@weblancaster I agree with you, although storyshots also uses jest & snapshot testing, just fully automated from your storybook stories 🙈 But yeah, let's improve tooling when we need it

**Comment by DanielMSchmidt** [comment]
I also added tshint as pretest step.

**Pull request merged**